toDate.js

```diff
@@ -1,4 +0,0 @@
-var date = process.argv[2];
-if (date !== 'null') {
-    console.log(new Date(date))
-}
```

Comment:
Build-bus was updated during Java 14 to only install the script, not the git module. This file was then missing